src/components/common/ColonyMembers/ColonyMembers.tsx

```diff
@@ -69,7 +69,7 @@ const ColonyMembers = () => {
           <TotalReputation selectedDomainId={selectedDomainId} />
           <MemberControls isRootOrAllDomains={isRootOrAllDomains} />
           <MembersFilter
-            onFilterChange={handleFilterChange}
+            handleFilterChange={handleFilterChange}
```

Comment:
Any reason for this change? I proposed this convention to easier distinguish between internal and external handlers:
https://github.com/JoinColony/ColonyFrontEndLivingStandard/commit/4a76e00551428ec638b02efd342b24512bd541d7

Reply:
Interesting. I had interpreted the rule as: "use 'handle' as the prefix for event handlers", as opposed to: "use 'handle' as the prefix for event handlers _and_ use 'on' as the prefix for all props that pass event handlers". 

That's perhaps worth clarifying in the entry in the living standard. 

We should also add this to help enforce it: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md

And create an issue to go and fix it everywhere it's not being adhered to, since I'm pretty sure it's not being adhered to generally.

Reply:
Also, wouldn't this only apply to native events? `onFilterChange` is not an actual event, it wraps the `onChange` prop of the `Select` component (which also not a native event). 

That's how I've interpret your distinction between "internal" and "external" event handlers but I might be misreading that.

Reply:
The idea behind the rule in living standard was the same as the rule you linked, that is to enforce all event handlers parsed as prop to start with `on`, and the handlers defined within the component file to start with `handle`. The benefits I see are that it's instantly clear what type of handler you're dealing with, and whenever you need to write code like this:
```js
const handleChange = () => {
  // call handler passed as prop
  onChange();
};
```
If that rule wasn't followed, you'd most likely need to find some alternative naming, which isn't great.

I tried playing around with enabling the eslint rule but it seems like it catches too many cases, e.g.:
```js
  const { updateWallet } = useAppContext();

   <ActionButton
     onSuccess={updateWallet}
   />
```
Yields an error because it expects `updateWallet` to start with `handle` which doesn't feel justified for such a trivial case. The crucial thing is that the `onSuccess` prop is named appropriately.

Thanks for highlighting the potential confusion, I modified the section in living standards: https://github.com/JoinColony/ColonyFrontEndLivingStandard/pull/16